### ISSUE TITLE
cursor won't stop blinking

### ISSUE THREAD

**Issue by nwaywood** [rhysd/neovim-component#28]
I'm not sure if its a bug or I'm doing something wrong, but I have tried 

```
<dom-module id="nyaovim-app">
  <template>
    <style>
      /* CSS configurations here */
    </style>

    <!-- Component tags here -->
    <neovim-editor id="nyaovim-editor" 
		argv$="[[argv]]" 
		font-size="14" 
		font="Fura Mono for Powerline Plus Nerd File Types,monospace"></neovim-editor>
  </template>
</dom-module>
<script>
	const neovim_element = document.getElementById('neovim');
	neovim_element.editor.blink_cursor = false;
</script>
```

and

```
<dom-module id="nyaovim-app">
  <template>
    <style>
      /* CSS configurations here */
    </style>

    <!-- Component tags here -->
    <neovim-editor id="nyaovim-editor" 
		argv$="[[argv]]" 
		font-size="14" 
        blink-cursor="false"
		font="Fura Mono for Powerline Plus Nerd File Types,monospace"></neovim-editor>
  </template>
</dom-module>
```

but the cursor doesn't stop blinking in normal mode.

**Comment by rhysd** [comment]
Hmm, `blink-cursor="false"` should stop blinking cursor.  I need to investigate.

**Comment by nwaywood** [comment]
@rhysd Thanks, and just to clarify, I am referring to the cursor blinking in `normal mode`.

**Comment by rhysd** [comment]
Please use `no-blink-cursor` attribute instead as below:

```html
<neovim-editor id="nyaovim-editor" 
    argv$="[[argv]]" 
    font-size="14" 
    no-blink-cursor
    font="Fura Mono for Powerline Plus Nerd File Types,monospace"></neovim-editor>
```

I renamed the attribute because HTML element's attribute can't have default `true` value.

https://github.com/Polymer/polymer/issues/1812

I'll ship this fix in v0.7.0.
